frontend/app/shared/components/schema-category.component.ts

```diff
@@ -49,13 +51,19 @@ export class SchemaCategoryComponent implements OnChanges {
     }
 
     public ngOnChanges() {
+        this.visibleCount = this.getCount(this.schemaCategory);
         if (this.schemaCategory.schemas.length < this.schemaCategory.schemaTotalCount) {
             this.isCollapsed = false;
         } else {
             this.isCollapsed = this.localStore.getBoolean(this.configKey());
         }
     }
 
+    private getCount(category: SchemaCategory): number {
```

Comment:
This method replaced the previous `emit` which allowed the controls to update either other when they filtered, but now the filtering is all in the state that's not needed any more 👍